scss/civicrm/common/_base.scss

```diff
@@ -149,6 +149,12 @@
     box-shadow: $box-shadow;
     margin-bottom: 20px;
 
+    // Sometimes there are .dataTables_wrapper > .dataTables_wrapper
+    // blocks that produce double shadows
```

Comment:
Add `.` in the end.

Reply:
Please check this at other places in the css code.

Reply:
Done.